index.html

```diff
@@ -86,13 +85,13 @@ <h6>Registration now Open!</h6>
         <div class="container">
             <div class="row">
             	<div class="FAQ_fx">
-				    	<span1 class="text-effect">faq</span1>
+				    	<span1 class="text-effect">FAQ</span1>
 				</div>
                 <div class="col-md-5 hidden-sm hidden-xs">
                 	
-                    <div class="faq-left-thumb">
-                        <img src="assets/img/icon/info.svg" alt="sponsor logo">
-                    </div>
+<!--                     <div class="faq-left-thumb">
+                        <img src="assets/img/bg/faqanimation.gif" alt="sponsor logo">

```

Comment:
Before merging, make sure that this gif doesn't infinitely loop, otherwise it's too distracting. can use any gif editor. check this link https://filmora.wondershare.com/animated-gif/how-to-remove-gif-loop.html